### PR TITLE
Fixed issues in iPad landscape

### PR DESCRIPTION
**Pull request by BenchR267**
Fixes issues when started in landscape. Check https://github.com/hyperoslo/ImagePicker/issues/100

### CODE REVIEW

File: Source/CameraView/CameraView.swift [hunk 1]
```diff
@@ -256,27 +258,6 @@ class CameraView: UIViewController {
     delegate?.imageToLibrary()
   }
 
-  func cropImage(image: UIImage) -> UIImage {
-    guard let imageReference = CGImageCreateWithImageInRect(image.CGImage,
-      CGRect(x: 0, y: 0, width: image.size.height - 285, height: image.size.width)) else { return UIImage() }
```
Comment (BenchR267) [inline Source/CameraView/CameraView.swift]: Anyway: why was here UIImage() returned instead of the input parameter?

**Comment by BenchR267** [comment]
Update: I also added a configuration flag to set if the collectionView should be hide itself while the photo is being taken. The default behaviour is unchanged.

**Comment by RamonGilabert** [comment]
For some reason Travis doesn't pass. Could you check that @BenchR267?

**Comment by BenchR267** [comment]
`$ brew install carthage
Error: carthage-0.11 already installed`

Was the error why it failed.. I don't know how the CI configuration is done but that should not cause an error. Is it possible to just restart the build @RamonGilabert?

**Comment by zenangst** [comment]
@BenchR267 try removing `- brew install carthage` from `.travis.yml` and I think it should work.

**Comment by BenchR267** [comment]
Seems to work now, thanks 👍 I also merged the master to resolve merge conflicts.

**Comment by zenangst** [comment]
@BenchR267 

tumblr_noqm39bgem1u0ci4io1_500

**Pull request merged**